float64 left_measured_travel # Distance traveled by left wheel in m
float64 right_measured_travel # Distance traveled by right wheel in m

float64 left_measured_vel # Left wheel velocity in m/s
float64 right_measured_vel # Right wheel velocity in m/s